lms/templates/instructor/instructor_dashboard_2/data_download.html

```diff
@@ -35,6 +35,10 @@
 
     <p><input type="button" name="list-may-enroll-csv" value="${_("Download a CSV of learners who can enroll")}" data-endpoint="${ section_data['get_students_who_may_enroll_url'] }" data-csv="true"></p>
 
+    %if settings.FEATURES.get('ENABLE_PROCTORED_EXAMS'):
```

Comment:
I'd prefer that we perform all logical checks in the view logic for determining whether or not to display certain UI components.  So in this case you'd move the "if settings.FEATURES..." check into the view and add a boolean flag to the context such as "show_generate_proctored_exam_report_button".